superbuild/projects_modules/eigen.cmake

```diff
@@ -0,0 +1,71 @@
+function(eigen_project)
+
+set(ep eigen)
+
+###############################################################################
+## List the dependencies of the project
+###############################################################################
+
+list(APPEND ${ep}_dependencies "")
+  
+###############################################################################
+## Prepare the project
+############################################################################### 
+
+EP_Initialisation(${ep}
+  USE_SYSTEM OFF 
+  BUILD_SHARED_LIBS OFF
+  REQUIRED_FOR_PLUGINS ON
+  )
+
+
+if (NOT USE_SYSTEM_${ep})
+
+## #############################################################################
+## Define repository for sources
+## #############################################################################
+
+set(git_url ${GITHUB_PREFIX}eigenteam/eigen-git-mirror)
+if(WIN32)
+  set(git_tag 9e97af7de76716c99abdbfd4a4acb182ef098808)
+else(WIN32)
```

Comment:
cf https://cmake.org/cmake/help/latest/command/if.html you can remove the `(WIN32)` here and line 33 below, it would be clearer